Gems/Atom/Utils/Code/Include/Atom/Utils/ImGuiGpuProfiler.inl

```diff
@@ -86,6 +89,96 @@ namespace AZ
                 }
                 drawList->AddText(font, font->FontSize, pos, ImGui::GetColorU32(ImGuiCol_Text), text, nullptr, size.x);
             }
+
+            inline static AZStd::string GetImageBindStrings(AZ::RHI::ImageBindFlags imageBindFlags)
+            {
+                // FIXME: any more elegant way to pull binding strings from the enum? Maybe through RTTI? ReflectSystemComponent.cpp L262
+                AZStd::string imageBindStrings;
+                if (AZ::RHI::CheckBitsAll(imageBindFlags, AZ::RHI::ImageBindFlags::Color))
+                {
+                    imageBindStrings.append("Color,");
+                }
+                else if (AZ::RHI::CheckBitsAll(imageBindFlags, AZ::RHI::ImageBindFlags::ShaderRead))
+                {
+                    imageBindStrings.append("ShaderRead,");
+                }
```

Comment:
This pattern is really ugly 😆 . Does anyone have ideas on how to make this more elegant? I'm trying to convert bitpacked enums to their string representation

Reply:
Yes. AZ_ENUM